pageserver/src/walredo.rs

```diff
@@ -547,6 +547,9 @@ impl PostgresRedoProcess {
         let stdin = &mut self.stdin;
         let stdout = &mut self.stdout;
 
+        // Buffer the writes to avoid a lot of small syscalls.
+        let mut buffered_stdin = tokio::io::BufWriter::new(&mut *stdin);
```

Comment:
Consider getting rid of `stdin` and just taking `&mut self.stdin` as the arg

Reply:
Good point. And actually, if we rename 'buffered_stdin' to 'stdin', this turns into a one-line patch.

Reply:
Nice!